mw_redis_cache.go

```diff
@@ -17,8 +17,7 @@ import (
 )
 
 const (
-	upstreamCacheHeader    = "x-tyk-cache-action-set"
-	upstreamCacheTTLHeader = "x-tyk-cache-action-set-ttl"
+	upstreamCacheHeader = "x-tyk-cache-action-set"
```

Comment:
Nit, but I'd keep the constants as they were - next to each other. They're very closely related, so it's useful.

Reply:
but is no longer a constant. We set the default to `"x-tyk-cache-action-set"` and change it if header name has been changed.

Reply:
Sure, but you could still have it there under `upstreamCacheTTLHeaderDefault` or something similar.